monitoring/statuscake/variables.tf

```diff
@@ -25,3 +25,21 @@ variable "confirmation" {
   description = "Retry the check when an error is detected to avoid false positives and micro downtimes"
   default     = 2
 }
+
+# Heartbeat check variables
+variable "name" {
+  description = "Name of the heartbeat check"
+  type        = string
```

Comment:
Can we pass a list, like the other statuscake checks? And default to [] to make it optional